CMakeLists.txt

```diff
@@ -1295,6 +1295,7 @@ if (CONFIG_MCUBOOT AND CONFIG_MCUBOOT_BUILD_S1_VARIANT)
 
   add_custom_target(
     s1_image_hex
+    ALL
```

Comment:
Should we include this as a revert + reapply of this commit: bfd1ae364c
and then include this change as part of the reapply commit ?
I believe this would minimize the numbers of [nrf noup] commit to handle during an upmerge.

@mbolivar guess as the master of upmerges you have an opinion on this.

Reply:
I'm unable to tell if that makes sense or not.

Reply:
@SebastianBoe -- I think we agreed yesterday that we'd work together to get this in and then clean it up at rebase time. Is that right? If so, I think you two should just do something that feels reasonable to you and we can decide what happens later.

Reply:
@mbolivar @SebastianBoe Agreed. Let's keep as is for now.